README.md

```diff
@@ -597,6 +597,56 @@ These are tips to help you with a successful project workflow
    }
    ```
 
+1. You can use [`Create nexus types`](https://github.com/AhmedElywa/create-nexus-type) tool to generate `objectType` , `queryType` and `mutationType` from your `schema.prisma` file for every model.
```

Comment:
Can you put this section under like so:

```md
### Community

#### `create-nexus-type`

...
```

Reply:
please can you share where i can put exactly here 
![image](https://user-images.githubusercontent.com/4535142/70554301-8ab5ec00-1b85-11ea-8539-f377c027df70.png)

Reply:
A new section, after `Reference`, called `Community`, please.

Reply:
Ok done